### ISSUE TITLE
Unable to exclude some log warnings

### ISSUE THREAD

**Issue by memeplex** [James-Yu/LaTeX-Workshop#4326]
> _Please fill the following fields with a star (*) and provide as much related information as possible._

## Pre-checks*
> _Please change the following `[ ]` to `[x]` for confirmation._
- [x] The issue has not been reported in this repository.
- [x] The issue remains after disabling all other extensions and restarting Visual Studio Code.
- [x] The [FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) cannot address the issue.
- [x] The issue is not related to compiling a document, or the document can be successfully compiled in the OS terminal but not in Visual Studio Code with this extension.

## Environment*

- Operating System: macOS Sonoma 14.4.1
- Visual Studio Code Version: 1.91.1 
- LaTeX Workshop Version: v10.1.0
- TeX Distribution Version: TeX Live 2024

## The Issue*

Despite adding:

```
  "latex-workshop.message.latexlog.exclude": [
    [...]
    ".*destination with the same identifier"
  ],
```

I'm unable to exclude:

<img width="484" alt="image" src="https://github.com/user-attachments/assets/04d4d020-6209-4c78-8633-556c21195061">

which according to the log must be coming from one of:

```
LaTeX Warning: Label `wip' multiply defined.
Label `wip' multiply defined
pdfTeX warning (ext4): destination with the same identifier (name{wip}) has been already used, duplicate ignore
```

The first two I've already filtered out, so the remaining one is the third.

Since the message in the Problems panel has more structure than usual, I believe that LW may be doing some processing of the log that circumvents the exclusion mechanism.

### LaTeX Workshop Output*
> _Please paste the whole log messages below, not parts of ones. The log should start with `New log placeholder %WS1% registered`._
```
[19:25:11.067][Logger] New log placeholder %WS1% registered for /Users/carlos/Documents/UCEMA/POOD .
[19:25:11.067][Extension] Initializing LaTeX Workshop.
[19:25:11.176][Build][Recipe] Set $LATEXWORKSHOP_DOCKER_LATEX: ""
[19:25:11.176][Build][Recipe] Set $LATEXWORKSHOP_DOCKER_PATH: "docker"
[19:25:11.207][Server] Creating LaTeX Workshop http and websocket server.
[19:25:11.273][Format][Bib] Bibtex format config: {"tab":"  ","case":"lowercase","left":"{","right":"}","trailingComma":false,"sort":["key"],"alignOnEqual":true,"sortFields":false,"fieldsOrder":[],"firstEntries":["string","xdata"]}
[19:25:11.306][Server] Server successfully started: {"address":"127.0.0.1","family":"IPv4","port":64487} .
[19:25:11.344][Extension] Extension root: /Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0
[19:25:11.344][Extension] $PATH: /Users/carlos/Documents/Util:/Users/carlos/.local/bin:/Users/carlos/.venvs/base/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
[19:25:11.344][Extension] $SHELL: /opt/homebrew/bin/bash
[19:25:11.345][Extension] $LANG: en_US.UTF-8
[19:25:11.345][Extension] $LC_ALL: en_US.UTF-8
[19:25:11.345][Extension] process.platform: darwin
[19:25:11.345][Extension] process.arch: arm64
[19:25:11.345][Extension] vscode.env.appName: Visual Studio Code
[19:25:11.345][Extension] vscode.env.remoteName: undefined
[19:25:11.345][Extension] vscode.env.uiKind: 1
[19:25:11.358][Config] latex-workshop.latex.outDir: "/tmp/lws-aux" .
[19:25:11.358][Config] latex-workshop.latex.texDirs: ["~/Documents/Util"] .
[19:25:11.360][Config] latex-workshop.latex.autoBuild.run: "never" .
[19:25:11.362][Config] latex-workshop.view.pdf.invertMode.enabled: "auto" .
[19:25:11.362][Config] latex-workshop.view.pdf.invert: 0.9 .
[19:25:11.363][Config] latex-workshop.synctex.indicator: "circle" .
[19:25:11.371][Config] latex-workshop.message.latexlog.exclude: ["Package fontawesome5 Warning: The requested icon","Label.*wip.*multiply defined",".*destination with the same identifier"] .
[19:25:11.372][Config] latex-workshop.hover.preview.scale: 1.15 .
[19:25:11.373][Config] latex-workshop.hover.preview.mathjax.extensions: ["verb"] .
[19:25:11.379][Extension] Trigger characters for intellisense of LaTeX documents: ["\\",",","{","}"]
[19:25:11.380][Root] Current workspace folders: ["file://%WS1%"]
[19:25:11.380][Extension] LaTeX Workshop initialized.
[19:25:11.380][Root] Current workspaceRootDir: file://%WS1% .
[19:25:11.385][Server] valdOrigin is http://127.0.0.1:64487
[19:25:11.400][Viewer][Panel] Internal PDF viewer at http://127.0.0.1:64487/viewer.html?file=pdf..ZmlsZSUzQSUyRiUyRiUyRnRtcCUyRmx3cy1hdXglMkZiYXNlLnBkZg .
[19:25:11.401][Cacher][Watcher] Watched /tmp/lws-aux/base.pdf with a new .pdf watcher on /tmp/lws-aux .
[19:25:11.401][Event] FILE_WATCHED: "/tmp/lws-aux/base.pdf"
[19:25:11.612][Root] Found files that might be root, choose the first one: %WS1%/base.tex,%WS1%/clase1.tex .
[19:25:11.612][Root] Root file changed: from %WS1%/base.tex to %WS1%/base.tex, langID latex . Refresh dependencies
[19:25:11.612][Event] ROOT_FILE_CHANGED: "%WS1%/base.tex"
[19:25:11.612][Cacher][Watcher] Reset.
[19:25:11.612][Cacher][Watcher] Reset.
[19:25:11.613][Cacher] Adding %WS1%/base.tex .
[19:25:11.613][Cacher][Watcher] Watched %WS1%/base.tex with a new .* watcher on %WS1% .
[19:25:11.614][Event] FILE_WATCHED: "%WS1%/base.tex"
[19:25:11.614][Cacher] Caching %WS1%/base.tex .
[19:25:11.614][Event] ROOT_FILE_SEARCHED
[19:25:11.617][Cacher] Updated inputs of %WS1%/base.tex .
[19:25:11.618][Cacher] Parse LaTeX AST: %WS1%/base.tex .
[19:25:12.311][Viewer] Handle data type: open
[19:25:12.478][Server] Preview PDF file: file:///tmp/lws-aux/base.pdf
[19:25:12.545][Root] Current workspace folders: ["file://%WS1%"]
[19:25:12.545][Root] Found root file from active editor: %WS1%/base.tex
[19:25:12.546][Root] Keep using the same root file: %WS1%/base.tex
[19:25:12.546][Event] ROOT_FILE_SEARCHED
[19:25:12.640][Viewer] Handle data type: loaded
[19:25:12.640][Event] VIEWER_PAGE_LOADED
[19:25:13.674][Cacher] Parsed LaTeX AST in 2056.50 ms: %WS1%/base.tex .
[19:25:13.675][File] Calling kpsewhich to resolve beamer.cls .
[19:25:14.012][File] kpsewhich returned with '/opt/homebrew/Cellar/texlive/20240312/share/texmf-dist/tex/latex/beamer/beamer.cls'.
[19:25:14.027][Cacher] Updated elements in 352.42 ms: %WS1%/base.tex .
[19:25:14.028][Event] FILE_PARSED: "%WS1%/base.tex"
[19:25:14.030][Structure] Structure force updated with 0 root sections for %WS1%/base.tex .
[19:25:14.030][Event] STRUCTURE_UPDATED
[19:25:14.030][Structure] Structure updated with 3 root sections for %WS1%/base.tex .
[19:25:14.030][Structure] Structure updated with 3 root sections for %WS1%/base.tex .
[19:25:14.031][Event] STRUCTURE_UPDATED
[19:25:14.031][Event] STRUCTURE_UPDATED
[19:25:14.041][Cacher] Parsing .fls /tmp/lws-aux/base.fls .
[19:25:14.123][Cacher] Found .aux /private/tmp/lws-aux/base.aux from .fls /tmp/lws-aux/base.fls , parsing.
[19:25:14.124][Cacher] Parsed .aux /private/tmp/lws-aux/base.aux .
[19:25:14.124][Cacher] Parsed .fls /tmp/lws-aux/base.fls .
[19:25:15.748][Event] STRUCTURE_UPDATED
[19:25:18.897][Commander] BUILD command invoked.
[19:25:18.897][Build] The document of the active editor: file://%WS1%/base.tex
[19:25:18.897][Build] The languageId of the document: latex
[19:25:18.898][Root] Current workspace folders: ["file://%WS1%"]
[19:25:18.898][Root] Found root file from active editor: %WS1%/base.tex
[19:25:18.899][Root] Keep using the same root file: %WS1%/base.tex
[19:25:18.899][Event] ROOT_FILE_SEARCHED
[19:25:18.899][Event] STRUCTURE_UPDATED
[19:25:18.899][Build] Building root file: %WS1%/base.tex
[19:25:18.899][Build][Recipe] Build root file %WS1%/base.tex
[19:25:18.918][Extension] onDidSaveTextDocument triggered: file://%WS1%/base.tex
[19:25:18.920][Build][Recipe] Preparing to run recipe: latexmk.
[19:25:18.921][Build][Recipe] Prepared 1 tools.
[19:25:18.936][Build][Recipe] outDir: /tmp/lws-aux .
[19:25:18.937][Build] Recipe step 1 The command is latexmk:["-synctex=1","-interaction=nonstopmode","-file-line-error","-pdf","-outdir=/tmp/lws-aux","%WS1%/base"].
[19:25:18.937][Build] env: {}
[19:25:18.937][Build] root: %WS1%/base.tex
[19:25:18.937][Build] cwd: %WS1%
[19:25:18.939][Build] LaTeX build process spawned with PID 30402.
[19:25:19.070][Cacher][Watcher] "change" emitted on %WS1%/base.tex.
[19:25:19.071][Cacher] Caching %WS1%/base.tex .
[19:25:19.071][Event] FILE_CHANGED: "%WS1%/base.tex"
[19:25:19.072][Cacher] Updated inputs of %WS1%/base.tex .
[19:25:19.072][Cacher] Parse LaTeX AST: %WS1%/base.tex .
[19:25:19.117][Cacher] Parsed LaTeX AST in 44.81 ms: %WS1%/base.tex .
[19:25:19.117][File] kpsewhich cache hit on beamer.cls: /opt/homebrew/Cellar/texlive/20240312/share/texmf-dist/tex/latex/beamer/beamer.cls .
[19:25:19.124][Cacher] Updated elements in 6.40 ms: %WS1%/base.tex .
[19:25:19.124][Event] FILE_PARSED: "%WS1%/base.tex"
[19:25:19.125][Structure] Structure force updated with 3 root sections for %WS1%/base.tex .
[19:25:19.125][Event] STRUCTURE_UPDATED
[19:25:19.983][Cacher][Watcher] "change" emitted on /tmp/lws-aux/base.pdf after polling for 252 ms.
[19:25:19.984][Event] FILE_CHANGED: "/tmp/lws-aux/base.pdf"
[19:25:20.505][Cacher][Watcher] "change" emitted on /tmp/lws-aux/base.pdf after polling for 252 ms.
[19:25:20.505][Event] FILE_CHANGED: "/tmp/lws-aux/base.pdf"
[19:25:21.504][Parser][TexLog] Logged 3 messages.
[19:25:21.505][Build] Finished a step in recipe with PID 30402.
[19:25:21.505][Build] Successfully built %WS1%/base.tex .
[19:25:21.505][Event] BUILD_DONE
[19:25:21.505][Viewer] Call refreshExistingViewer: "/tmp/lws-aux/base.pdf" .
[19:25:21.505][Viewer] Refresh PDF viewer: /tmp/lws-aux/base.pdf
[19:25:21.509][Cacher] Parsing .fls /tmp/lws-aux/base.fls .
[19:25:21.522][Cacher] Found .aux /private/tmp/lws-aux/base.aux from .fls /tmp/lws-aux/base.fls , parsing.
[19:25:21.523][Cacher] Parsed .aux /private/tmp/lws-aux/base.aux .
[19:25:21.523][Cacher] Parsed .fls /tmp/lws-aux/base.fls .
[19:25:21.524][Server] Preview PDF file: file:///tmp/lws-aux/base.pdf
[19:25:21.593][Viewer] Handle data type: loaded
[19:25:21.593][Event] VIEWER_PAGE_LOADED
```

### Developer Tools Console
> _Please paste the whole log messages below, not parts of ones. This console logs can sometimes be very important in many cases. To access the log, click `Help` -> `Toggle Developer Tools` -> `Console`._
```
log.ts:419  INFO Started local extension host with pid 30019.
log.ts:439   ERR Extension 'ms-toolsai.datawrangler' wants API proposal 'debugFocus' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
TMScopeRegistry.ts:46 Overwriting grammar scope name to file mapping for scope text.tex.
Old grammar file: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/extensions/latex/syntaxes/TeX.tmLanguage.json.
New grammar file: file:///Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0/syntax/TeX.tmLanguage.json
register @ TMScopeRegistry.ts:46
TMScopeRegistry.ts:46 Overwriting grammar scope name to file mapping for scope text.tex.latex.
Old grammar file: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/extensions/latex/syntaxes/LaTeX.tmLanguage.json.
New grammar file: file:///Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0/syntax/LaTeX.tmLanguage.json
register @ TMScopeRegistry.ts:46
TMScopeRegistry.ts:46 Overwriting grammar scope name to file mapping for scope text.bibtex.
Old grammar file: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/extensions/latex/syntaxes/Bibtex.tmLanguage.json.
New grammar file: file:///Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0/syntax/Bibtex.tmLanguage.json
register @ TMScopeRegistry.ts:46
TMScopeRegistry.ts:46 Overwriting grammar scope name to file mapping for scope text.tex.markdown_latex_combined.
Old grammar file: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/extensions/latex/syntaxes/markdown-latex-combined.tmLanguage.json.
New grammar file: file:///Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0/syntax/markdown-latex-combined.tmLanguage.json
register @ TMScopeRegistry.ts:46
TMScopeRegistry.ts:46 Overwriting grammar scope name to file mapping for scope source.cpp.embedded.latex.
Old grammar file: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/extensions/latex/syntaxes/cpp-grammar-bailout.tmLanguage.json.
New grammar file: file:///Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0/syntax/cpp-grammar-bailout.tmLanguage.json
register @ TMScopeRegistry.ts:46
webviewElement.ts:482 An iframe which has both allow-scripts and allow-same-origin for its sandbox attribute can escape its sandboxing.
mountTo @ webviewElement.ts:482
TMScopeRegistry.ts:46 Overwriting grammar scope name to file mapping for scope text.tex.
Old grammar file: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/extensions/latex/syntaxes/TeX.tmLanguage.json.
New grammar file: file:///Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0/syntax/TeX.tmLanguage.json
register @ TMScopeRegistry.ts:46
TMScopeRegistry.ts:46 Overwriting grammar scope name to file mapping for scope text.tex.latex.
Old grammar file: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/extensions/latex/syntaxes/LaTeX.tmLanguage.json.
New grammar file: file:///Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0/syntax/LaTeX.tmLanguage.json
register @ TMScopeRegistry.ts:46
TMScopeRegistry.ts:46 Overwriting grammar scope name to file mapping for scope text.bibtex.
Old grammar file: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/extensions/latex/syntaxes/Bibtex.tmLanguage.json.
New grammar file: file:///Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0/syntax/Bibtex.tmLanguage.json
register @ TMScopeRegistry.ts:46
TMScopeRegistry.ts:46 Overwriting grammar scope name to file mapping for scope text.tex.markdown_latex_combined.
Old grammar file: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/extensions/latex/syntaxes/markdown-latex-combined.tmLanguage.json.
New grammar file: file:///Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0/syntax/markdown-latex-combined.tmLanguage.json
register @ TMScopeRegistry.ts:46
TMScopeRegistry.ts:46 Overwriting grammar scope name to file mapping for scope source.cpp.embedded.latex.
Old grammar file: file:///Applications/Visual%20Studio%20Code.app/Contents/Resources/app/extensions/latex/syntaxes/cpp-grammar-bailout.tmLanguage.json.
New grammar file: file:///Users/carlos/.vscode/extensions/james-yu.latex-workshop-10.1.0/syntax/cpp-grammar-bailout.tmLanguage.json
register @ TMScopeRegistry.ts:46
log.ts:419  INFO [perf] Render performance baseline is 23ms
log.ts:439   ERR Unknown error while installing extension ms-toolsai.jupyter-renderers: Unknown: Unknown error while installing extension ms-toolsai.jupyter-renderers
    at V.installFromGallery (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:105:61282)
```

**Comment by James-Yu** [comment]
It's not related to log parsing. We have our own duplicate label detection scheme. You can disable it with `latex-workshop.check.duplicatedLabels.enabled`
